### ISSUE TITLE
[CI/CD]: Continue exploration of description list refactoring for all-claims accessibility checks

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8595]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This issue continues the work @Mottie started for refactoring the description lists in #7319 His comment to me in [PR #12408](https://github.com/department-of-veterans-affairs/vets-website/pull/12408) prompted this new issue ticket:

> @1Copenut If you don't mind, I'd like to address those other two additional problems in a different PR. This one was meant to handle the ReviewCardField (and the ObjectField, which is also a problem in the above screenshots you shared). Which only applies to the email, phone, address & bank account sections.

Including two screen shots below of the items this issue will focus on remediating.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## Environment
* `vets-website/src/applications/disability-benefits/all-claims/tests`
* `$ yarn test:e2e src/applications/disability-benefits/all-claims/tests`

## Acceptance Criteria

**Definition of done:**
- [ ] Front-end team member(s) have reviewed end-to-end tests for axe checks
- [ ] Front-end engineers / testers can open every accordion on the Review Your Results page, and have no axe `violations` appear in the plugin console.
- [ ] Rendered HTML for the description list and associated children passes a  validation check
- [ ] No axe `violations` appear in the plugin console. These will break the CI/CD build.

## WCAG or Vendor Guidance (optional)

* [Custom axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js)
* [VSP guidance on writing end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing)

## Screen Shots or Trace Logs
![Screen Shot 2020-04-29 at 10 13 28 AM](https://user-images.githubusercontent.com/934879/80719617-59e93b80-8ac1-11ea-8fd3-4d88c33f2e91.png)

---
![Screen Shot 2020-04-29 at 10 46 10 AM](https://user-images.githubusercontent.com/934879/80719635-6077b300-8ac1-11ea-8195-24f004689409.png)
